src/figuresClasses.ts

```diff
@@ -1,11 +1,119 @@
-export interface Figure {}
+type ShapeType = `triangle` | `circle` | `rectangle`;
+type ColorType = `red` | `green` | `blue`;
 
-export class Triangle implements Figure {}
+function checkSidesLength(...sides: number[]): boolean {
+  for (const side of sides) {
+    if (side <= 0) {
+      return false;
+    }
+  }
 
-export class Circle implements Figure {}
+  return true;
+}
+
+function roundToHundr(n: number): number {
+  return Number(parseInt(`${n * 100}`) / 100);
+}
+
+export interface Figure {
+  shape: ShapeType;
+  color: ColorType;
+
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  constructor(
```

Comment:
[CODE KNOWLEDGE] - You don't need to manually assign the constructor parameters to class properties if you are using the 'public' keyword, as TypeScript does this automatically.